src/CONST.ts

```diff
@@ -2972,6 +2972,50 @@ const CONST = {
             ANNUAL: 'yearly2018',
             PAYPERUSE: 'monthly2018',
         },
+        get PRICES() {
+            return {
+                [CONST.PAYMENT_CARD_CURRENCY.USD]: {
```

Comment:
I think we can use `this` keyword here instead of `CONST`

Reply:
When I use `this` instead of `CONST`, this typescript error occurs.

![Screenshot 2024-12-22 at 22 23 40](https://github.com/user-attachments/assets/cfee68e4-8082-495b-a4aa-413813ac4e4c)

Reply:
Yeah, you are right. I think this because `this` keyword refer to the current object `SUBSCRIPTION`, not to the global object.

```javascript
const CONST = {
  SUBSCRIPTION: {
    get PRICES() {
      // using `this` here refer to `SUBSCRIPTION` not to `CONST`
    }
  }
}

```

Reply:
Although it works fine, but honestly I didn't know what's the impact of using the same object inside itself? But I see it didn't use before on this file, so I prefer moving `PRICES` out `SUBSCRIPTION` and use `this`. WDYT?

Reply:
That's Okay.